src/appengine/libs/issue_management/monorail/__init__.py

```diff
@@ -234,10 +234,6 @@ def get_issue(self, issue_id):
 
     return Issue(monorail_issue)
 
-  def get_original_issue(self, issue_id):
-    """Retrieve the original issue object traversing the list of duplicates."""
-    return Issue(self._itm.get_original_issue(int(issue_id)))
```

Comment:
Probably i can remove some of these dead/duplicate functions in monorail part of code.
E.g. https://github.com/google/clusterfuzz/blob/master/src/appengine/libs/issue_management/monorail/issue_tracker_manager.py#L305. wdyt ? i can do this when i am back.

Reply:
Yep, sgtm!